Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -1731,7 +1731,7 @@
             "type": 3,
             "content": {
               "version": "KqlItem/1.0",
-              "query": "resources\r\n| where type =~ \"Microsoft.DataBoxEdge/dataBoxEdgeDevices\"\r\n| where id =~ \"{ase}\"\r\n| project [\"ASE\"]=id, [\"Resource Group\"] = strcat(\"/subscriptions/\", subscriptionId, \"/resourceGroups/\", resourceGroup), [\"Status\"]=properties.dataBoxEdgeDeviceStatus, [\"Version\"]=properties.deviceSoftwareVersion",
+              "query": "resources\r\n| where type =~ \"Microsoft.DataBoxEdge/dataBoxEdgeDevices\"\r\n| where id =~ \"{ase}\"\r\n| project [\"ASE\"]=id, [\"Resource Group\"] = strcat(\"/subscriptions/\", subscriptionId, \"/resourceGroups/\", resourceGroup), [\"Status\"]=properties.dataBoxEdgeDeviceStatus, [\"Version\"]=properties.deviceSoftwareVersion, Location = location, Serial = properties.serialNumber",
```

Comment:
If you set the formatting on the location column to "Location" it will automatically convert it to the long-form name of the region, e.g. "East US" for eastus
![image](https://github.com/microsoft/Application-Insights-Workbooks/assets/3382572/838a27c7-c58c-4aec-b990-d56ec7538c02)